### ISSUE TITLE
[ON HOLD] Transform Modules

### ISSUE THREAD

**Issue by jejjohnson** [jejjohnson/oceanbench#26]
> A suggested List of Transformation Modules. Depends on #15 

---
# Motivation

See API discussion in #15 

---
# TODO

* [ ] Spherical -> Cartesian - [Example with Astropy](https://docs.astropy.org/en/stable/api/astropy.coordinates.cartesian_to_spherical.html) | [Example with pyproj](https://climate-cms.org/posts/2021-11-02-pyproj-regrid.html) | [Example with pyproj](https://climate-cms.org/posts/2021-10-01-pyproj.html) [metpy Example](https://unidata.github.io/MetPy/latest/examples/gridding/Wind_SLP_Interpolation.html#sphx-glr-examples-gridding-wind-slp-interpolation-py)
* [ ] Normalization
* [ ] Standard Scaler - subtract mean, divide std
* [ ] Filtering, e.g. `gaussian`
* [ ] Noising - useful for added arbitrary noise
* [ ] DeNoising
* [ ] Climatology
* [ ] Domain Slicing
* [ ] Time Slicing
